### ISSUE TITLE
Merge Editor: A way to look through automatically resolved "conflicts"

### ISSUE THREAD

**Issue by Aerolfos** [microsoft/vscode#162039]
![image](https://user-images.githubusercontent.com/8443014/192606425-893e1ea8-51ba-461a-988f-93ff2451f092.png)
There are arrows to go up and down for unresolved conflicts. Useful, but there seems to be no equivalent feature to look through _resolved_ conflicts that were handled automatically.

![image](https://user-images.githubusercontent.com/8443014/192607036-b7895d50-5a65-474e-8089-f1900c0b9f3b.png)
Basically, a way to skip between _all_ checkboxes, both pre-checked and ones you previously checked (for a final look through the merged code). In an example with 0 conflicts remaining, the arrows do nothing, and there are no alternative arrows either.
